Add explicit handling of "empty" Tuples Bridge returns for missing Tuples

tatu-at-datastax opened the pull request:
**What this PR does**:

Adds explicit handling (and tests) for cases where Bridge returns "empty" Tuple (that represents missing value for a Tuple column): will be converted to JSON `null`

**Which issue(s) this PR fixes**:
Fixes #2246

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

ivansenic reviewed (approved):
I am giving an approval, but I am not sure how the integration test is confirm this works now..

tatu-at-datastax commented:
> I am giving an approval, but I am not sure how the integration test is confirm this works now..

I added a case where new row is added without specifying `Tuple` value: this is then fetched.
Before fix it failed with exception reported; after fix does not.
Also added unit test for same conversion.

So there are tests that I think cover what is needed. But I think Cedric can validate via `curl` once this is merged.
